source/features/previous-version.tsx

```diff
@@ -7,12 +6,12 @@ import observe from '../helpers/selector-observer';
 import * as api from '../github-helpers/api';
 import GitHubURL from '../github-helpers/github-url';
 
-const getPastCommits = cache.function('previous-version', async (pathname: string): Promise<string[]> => {
```

Comment:
I dropped the cache because this would create a new cache item for every page visited, recreating the entire data every time you click on "previous" anyway.